### PR TITLE
Fix 5607

### LINKED ISSUE #5607: Setting 'wincolor' on finished terminal window causes Vim crash
**Describe the bug**
Setting or unsetting `wincolor` on finished terminal window causes Vim crash.
It seems that Vim doesn't crash if the terminal is running.

I understand that `wincolor` does not take effect for terminal windows, but I think Vim shouldn't crash.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Run `vim --clean`
2. `:terminal cmd.exe /C` (opens a terminal that immediately finish and leave opened)
3. `:set wincolor=`
4. Vim crashes

**Expected behavior**
Vim shouldn't crash.

**Environment (please complete the following information):**
 - Vim 8.2.235 (from vim/vim-win32-installer)
 - OS: Windows 10 1909 64bit
 - Terminal: Command Prompt and GUI

### PR DESCRIPTION
**Pull request by mattn**
Fixes #5607

### CODE REVIEW

File: src/testdir/test_terminal.vim [hunk 1]
```diff
@@ -2370,3 +2370,18 @@ func Test_terminal_in_popup()
   call StopVimInTerminal(buf)
   call delete('XtermPopup')
 endfunc
+
+func Test_issue_5607()
+  if !has('win32')
```
Comment (k-takata) [inline src/testdir/test_terminal.vim]: https://github.com/vim/vim/issues/5607#issuecomment-583945615
It seems that this is not Windows-specific.

Comment (mattn) [reply src/testdir/test_terminal.vim]: Fixed.

File: src/testdir/test_terminal.vim [hunk 2]
```diff
@@ -2372,9 +2372,6 @@ func Test_terminal_in_popup()
 endfunc
 
 func Test_issue_5607()
-  if !has('win32')
-    return
-  endif
   terminal cmd /c
```
Comment (k-takata) [inline src/testdir/test_terminal.vim]: This only works on Windows.

Comment (mattn) [reply src/testdir/test_terminal.vim]: Oh, i miss it.

**Pull request closed without merging**